README.md

```diff
@@ -401,7 +409,123 @@ via [additional environment variables](https://github.com/FIWARE/helm-charts/blo
 
 ### IAM components
 
-> (similar to the Access Node section)
+The [DOME IAM-Framework](https://github.com/DOME-Marketplace/iam-components) is a set of microservices, that enables
+users in the DOME ecosystem to authenticate into the [DOME Marketplace](https://dome-marketplace.org).
+
+#### Overview and subcomponents
+
+The DOME IAM-Framework consists of multiple open-source components. The components are not required to be used, as long
+as alternatives providing the same interfaces are used.
+
+The IAM-Framework consists of following components:
```

Comment:
WaltId is not listed below. Although we won't need it soon anymore, at the moment it is required. So we should also put a link here for consistency.